src/Arrays.php

```diff
@@ -445,4 +446,56 @@ public static function wrap(mixed $value, bool $wrapNull = true): array
 
         return [$value];
     }
+
+    /**
+     * Converts an array to a JSON string.
+     *
+     * @param array       $items
+     * @param string|null $failureMessage
+     *
+     * @return string
+     */
+    public static function toJson(array $items, ?string $failureMessage = null): string
+    {
+        if (count($items) === 0) {
+            throw new RuntimeException('Array is empty' . ($failureMessage === null ? '' : '. ' . $failureMessage));
+        }
+
+        try {
+            $json = json_encode($items, JSON_THROW_ON_ERROR);
+            if ($json === false) {
+                throw new RuntimeException('Unable to convert to Json' . ($failureMessage === null ? '' : '. ' . $failureMessage));
+            }
+        } catch (JsonException $e) {
+            throw new RuntimeException('Unable to convert to Json' . ($failureMessage === null ? $e : '. ' . $failureMessage));
+        }
+
+        return $json;
+    }
+
+    /**
+     * Converts a JSON string to an array.
+     *
+     * @param string      $jsonString
+     * @param string|null $failureMessage
+     *
+     * @return array
+     */
+    public static function fromJson(string $jsonString, ?string $failureMessage = null): array
+    {
+        if (strlen($jsonString) === 0) {
+            throw new RuntimeException('JSON string is empty' . ($failureMessage === null ? '' : '. ' . $failureMessage));
+        }
+
+        try {
+            $array = json_decode($jsonString, true, 512, JSON_THROW_ON_ERROR);
+            if ($array === null && json_last_error() !== JSON_ERROR_NONE) {
```

Comment:
Don't check for json_last_error (is not needed when using JSON_THROW_ON_ERROR)
use `return Assert::isArray(json_decode(....));`